docs/manual/mkdocs.yml

```diff
@@ -271,10 +273,11 @@ nav:
       - user-guide/analyzing/metadata.md
     - 'Workflow':
       - user-guide/workflow/index.md
-      - user-guide/workflow/batchediting.md
-      - user-guide/workflow/batchupdate-from-admin.md
-      - user-guide/workflow/batchupdate-xsl.md
-      - user-guide/workflow/batchupdate.md
+      - 'Updating a set of records':
+        - user-guide/workflow/batchupdate.md
```

Comment:
Maybe to change the title in `user-guide/workflow/batchupdate.md` to `Updating a set of records with batch update`, otherwise in the navigation bar are is displayed with the same title as the container: `Updating a set of records`

Reply:
@jodygarnett please check to review the previous comment and we can merge the pull request, thanks.